language/java/build-images.md

```diff
@@ -81,12 +81,10 @@ we would like to use for our application.
 ```dockerfile
 # syntax=docker/dockerfile:1
```

Comment:
Not linked, but we should also use the latest syntax version if we update the manual (maybe also backport to the blog post!)
`# syntax=docker/dockerfile:1.4`

Reply:
Thank you. One section containing this is shared with the other language-specific guides. Is it okay if I update this in a separate PR as it will require updates to the other guides?